spec/services/api/get_versioned_item_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Api::GetVersionedItem do
+  describe "#for_service" do
```

Comment:
These tests are testing the same thing right? Each method is just a wrapper around the `.for...` method? If we write tests for that method, then we can just test that the other methods call it with the correct `item_type`, and it saves the repetition? That'll make it easier if we add to this class in the future.

Reply:
Fixed it uses shared examples now